src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -435,7 +484,9 @@ def test_init_backwards_compatibility_with_initialize_with_channels(resource_nam
         ('Dev1/0', ''),
         ('Dev1/0,Dev1/1', None),
         ('Dev1/0,Dev2/1', None),
-        ('Dev1/0,Dev2/1', '')
+        ('Dev1/0,Dev2/1', ''),
+        (['Dev1/0', 'Dev1/1'], ''),  # construct with list
```

Comment:
Add a case where resource name is something like `Dev1/0-3`

Reply:
0e2734304b014bb6c7392e0134fa804d77611466